controllers/apiController.js

```diff
@@ -2,24 +2,111 @@ const fs = require('fs');
 const express = require('express');
 const router = express.Router();
 const bodyParser = require('body-parser')
+const MongoClient = require('mongodb').MongoClient;
+const ObjectID = require('mongodb').ObjectID;
 
 router.use(bodyParser.json());
 
+
+
+
+// First Callback Function to connect to database
+
+const connectAndFindAll = function (collection, cb) {
```

Comment:
be consistent with either using `functions` or `ES6 arrow methods` all the time

Reply:
also move this function outside apiController.js - `export` it and `require` it here

Reply:
Now, I see you actually have dbClient - you should reuse that module (maybe add possiibility to pass collection to it) and reuse it from here. Try to eliminate duplicate code as much as possible.